_config.yml

```diff
@@ -4,8 +4,8 @@ baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://www.amundsen.io" # the base hostname & protocol for your site, e.g. http://example.com
 
 #Links within the site
-documentation: "https://lyft.github.io/amundsen/"
-getstarted: "https://lyft.github.io/amundsen/"
+documentation: "https://www.amundsen.io/amundsen/"
+getstarted: "https://www.amundsen.io/amundsen/"
 youtube: "https://www.youtube.com/playlist?list=PL0UJdxehTNlKnGU_h7k2fzJyvAiufeh1U"
 github: "https://github.com/lyft/amundsen"
```

Comment:
Thanks for the change @jornh. I suppose the above link needs to be changed as well.